src/qt/bitcoingui.cpp

```diff
@@ -502,6 +492,13 @@ void BitcoinGUI::createMenuBar()
         window_menu->addAction(usedSendingAddressesAction);
         window_menu->addAction(usedReceivingAddressesAction);
     }
+#else
+    if (walletFrame) {
```

Comment:
Could code duplication be avoided?

Reply:
I dont see any nice, clean way if we want to keep current behavior. Not duplicating sending/receiving addresses would require duplicating check for macos.

Reply:
That said, it would save us a few symbols, though not any lines...

Reply:
The initial `#endif` line could be leaved in its place without adding the duplicated code, no?

Reply:
... I take it back, I was thinking about this when I was away from my computer.
Looks like I managed to decrease the line count by 5 by doing what I had in mind. Thanks for the idea.